charts/jaeger/templates/agent-sa.yaml

```diff
@@ -6,4 +6,8 @@ metadata:
   labels:
     {{- include "jaeger.labels" . | nindent 4 }}
     app.kubernetes.io/component: agent
+  {{- if .Values.agent.serviceAccount.annotations }}
+  annotations:
+  {{ toYaml .Values.agent.serviceAccount.annotations | indent 4 }}
+  {{- end }}
```

Comment:
nit: Could you adapt and follow the pattern from `helm create` to stay in sync?: https://github.com/helm/helm/blob/8be6bc076f3846dc15b2dd0153c3d64b527034e9/pkg/chartutil/create.go#L342-L345

Both work but I like to stay in sync with `helm create` as much as possible. If not no worries though.

Reply:
I change if to with. Tested and as you say - it works the same. I used it exactly from the link you added - is that ok?

Reply:
Thanks!